sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitCoverageResultParser.java

```diff
@@ -171,36 +168,32 @@ protected void saveCoverageMeasure(FileNode fileNode, List<String> unresolvedPat
     if (inputFile != null) {
       org.sonar.api.resources.File phpFile = org.sonar.api.resources.File.create(inputFile.relativePath());
       resolvedPaths.add(inputFile.relativePath());
-      saveCoverageLineHitsData(fileNode, inputFile, phpFile);
-
-      // Save uncovered statements (lines)
-      MetricsNode metrics = fileNode.getMetrics();
-      double totalStatementsCount = metrics.getTotalStatementsCount();
-      double uncoveredLines = totalStatementsCount - metrics.getCoveredStatements();
-      context.saveMeasure(phpFile, linesToCoverMetric, totalStatementsCount);
-      context.saveMeasure(phpFile, this.uncoveredLinesMetric, uncoveredLines);
+      saveCoverageLineHitsData(fileNode, inputFile, phpFile, context);
+
+      // Save uncovered statements (lines) no longer needed because coverage metrics are internally derived by the NewCoverage
     } else {
       unresolvedPaths.add(path);
     }
   }
 
-  private void saveCoverageLineHitsData(FileNode fileNode, InputFile inputFile, org.sonar.api.resources.File phpFile) {
+  private void saveCoverageLineHitsData(FileNode fileNode, InputFile inputFile, org.sonar.api.resources.File phpFile, SensorContext context) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "File"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)